### DISCUSSION TITLE
Cannot finish the compass calibration

### DISCUSSION THREAD

**Discussion by thiensu1122** [HefnySco/OBAL#10] [Q&A] [answered]
Hi HefnySco,

This is another problem that I found. In the Mission Planner -> compass calibration. I just cannot finish the calibration after it reaches 100%. it just restarts itself from 0% over and over again.

![image](https://user-images.githubusercontent.com/20256382/155869093-3fa784c6-f60d-4ace-8637-b4e4e42c4f88.png)

Do you know how to deal with this?

**Comment by HefnySco** [comment, marked as answer]
I usually use QGC as I use Linux. but It should not matter. Make sure that there is no magnetic field around your board while calibration, and rotate it all sides. This is how I do it and it works.
